modules/articles/client/articles.client.module.js

```diff
@@ -1,7 +1,7 @@
 (function (app) {
   'use strict';
 
-  app.registerModule('articles');
+  app.registerModule('articles', ['core']);
```

Comment:
I'm not sure if this forces Core to load first though. I know my wording above indicates that it does. But now that I think about it, all this does is add Core as a dependency; it's sort of ambiguous as to whether Core is indeed loading first, or not.

Perhaps a comment more along the lines of Core being required for special route handling? This aspect of module dependency feels weird to me. But I guess it's the intention of Angular's module dependency injection. I really feel like we should be able to tell Angular to load Core first, or into each module. Rather than injecting it into every single module definition. However, I guess that's a bit off topic :)